libraries/broker-legacy/CMakeLists.txt

```diff
@@ -318,6 +318,7 @@ function(generateBroker)
     PUBLIC
       zeek_agent_cxx_settings
       thirdparty_actorframework
+      thirdparty_caf_openssl
```

Comment:
If this target only requires OpenSSL, it is also possible to link against `thirdparty_openssl`.

Reply:
I think this should be the right dependency given this lib does not directly link against OpenSSL itself. Or am I missing something?